.github/workflows/daily-product-scan.yml

```diff
@@ -0,0 +1,61 @@
+on:
+  push
+  # schedule:
+  #   - cron: '0 12 * * 1-5'
```

Comment:
Can this be removed?

Reply:
No yet. I'm waiting to get an App Installation ID from GitHub Services. This new job won't run until I get it. Once I confirm everything it working, I'll remove that stuff.